src/main/java/io/netty/incubator/codec/http3/Http3ClientConnectionHandler.java

```diff
@@ -56,4 +76,16 @@ void initBidirectionalStream(ChannelHandlerContext ctx, QuicStreamChannel channe
         Http3CodecUtils.connectionError(ctx, Http3ErrorCode.H3_STREAM_CREATION_ERROR,
                 "Server initiated bidirectional streams are not allowed", true);
     }
+
+    @Override
+    void initUnidirectionalStream(ChannelHandlerContext ctx, QuicStreamChannel streamChannel) {
+        final Long maxTableCapacity = remoteControlStreamHandler.localSettings()
+                .get(HTTP3_SETTINGS_QPACK_MAX_TABLE_CAPACITY);
+        streamChannel.pipeline().addLast(
```

Comment:
I think we need to do something if maxTableCapacity is null or I am wrong ?

Reply:
`QpackEncoderHandler` handles `null` value for `maxTableCapacity`